Extension/src/LanguageServer/customProviders.ts

```diff
@@ -52,10 +52,10 @@ class CustomProviderWrapper implements CustomConfigurationProvider1 {
             valid = !!(this.provider.extensionId && this.provider.dispose);
         }
         if (valid && this._version > Version.v1) {
-            valid = !!(this.provider.canProvideBrowseConfiguration && this.provider.provideBrowseConfiguration);
+            valid = !!(this.provider.provideBrowseConfiguration);
         }
         if (valid && this._version > Version.v2) {
-            valid = !!(this.provider.canProvideBrowseConfigurationsPerFolder && this.provider.provideFolderBrowseConfiguration);
+            valid = !!(this.provider.provideFolderBrowseConfiguration);
```

Comment:
Why is this changing?  Both methods must exist on the v2 interface.

Reply:
Because it generates an error. this.provider.canProvideBrowseConfigurationsPerFolder is always defined: `This condition will always return true since this function is always defined. Did you mean to call it instead?`.

Reply:
Then you should tell the linter to ignore it for now.  `this.provider` is typed, but at this point it is being treated as `any`.  We should make the provider `unknown` until the fields are validated, but that can be in another PR.  (assign it to me, I'll do it real quick)

Reply:
@bobbrow Sure, I just undid the change...what do I assign to you? Can you just create a PR?

Reply:
I'll add the tslint ignore to your branch and fix it in my own afterwards

Reply:
Actually, there was another quick fix to avoid the lint error.  I pushed it to your branch.